### DISCUSSION TITLE
LF thoughts

### DISCUSSION THREAD

**Discussion by ghost** [Prodigy-Hacking/ProdigyMathGameHacking#1236] [(D) Developer]
![image](https://user-images.githubusercontent.com/61570792/147611112-6334a583-e088-43ef-9a00-a333583f4e2f.png)

See [the document](https://torrentfreak.com/images/epicaproval.pdf) for how a minor was sued: basically, his parent was held accountable.

Copyright infringement might apply because hacks.prodigyhacking.com hosts a _copy_ of Prodigy's code, but I don't know if it counts as a copy because it isn't stored anywhere.

In the Epics Games case, they also sued for breach of contract (the ToS of a site is a contract, I think). Prodigy may be able to trick the player's parent into agreeing to something that's like "you can't let your kid make hacks", but that might take them too much effort (which they hate).

And, yes, there is money to sue over (btw, you don't need to make money to be sued - you just need to cause them to lose money): money is lost because Prodigy hacks let you get free membership, which is supposed to cost money.

It also disengages players because a lower level of challenge and perceived skill in a game leads to boredom or apathy unless they are very skilled (i.e. low challenge & high skill = relaxation), but I doubt they would consider themselves skilled or challenged if they get everything for free ([source](https://en.wikipedia.org/wiki/File:Challenge_vs_skill.svg)).

That being said, I don't think shutting down the hacks is the right answer. If that's done, Prodigy will never be held accountable for letting this whole mess escalate and never fixing the problem Natsumi raised.

I hope Prodigy doesn't shut this down instead of fixing the problem, so please give me your thoughts.

**Comment by 00100000** [comment]
If Prodigy were to actually take legal action against this project, regardless of the strength of their case, as long as they somewhat have one, we'd be forced to shut down. Lawyers are expensive.

**Reply by ghost** [reply]
Yes, but the court does throw out frivolous cases (i.e. cases with no merit) before the defendant has to start paying for anything.

**Comment by donaldli2020** [comment]
I think we should use the variables inside the game instead of making our own.
So then it probably wouldn't be "copyright infringement."
